### PR TITLE
Revamp docs for Quantized MobileNetV3

### PR DESCRIPTION
**Pull request by abhi-glitchhg**
Related to #5987 

cc : @NicolasHug

### CODE REVIEW

File: torchvision/models/quantization/mobilenetv3.py [hunk 1]
```diff
@@ -194,18 +194,33 @@ def mobilenet_v3_large(
...
+    MobileNetV3 model from
+    `Searching for MobileNetV3 <https://arxiv.org/abs/1905.02244>`_.
 
-    Note that quantize = True returns a quantized model with 8 bit
-    weights. Quantized models only support inference and run on CPUs.
-    GPU inference is not yet supported
+    .. note::
+        Note that `quantize = True` returns a quantized model with 8 bit
+        weights. Quantized models only support inference and run on CPUs.
+        GPU inference is not yet supported
```
Comment (abhi-glitchhg) [inline torchvision/models/quantization/mobilenetv3.py:203]: I think we should highlight this note in other models' docstrings too.

**Comment by abhi-glitchhg** [comment]
[generated docs](https://output.circle-artifacts.com/output/job/42b189ba-c859-4117-8ec4-049de90f35f6/artifacts/0/docs/models/mobilenetv3_quant.html?highlight=quantized) for mobilenet v3

File: torchvision/models/quantization/mobilenetv3.py [hunk 2]
```diff
@@ -194,18 +194,33 @@ def mobilenet_v3_large(
     **kwargs: Any,
 ) -> QuantizableMobileNetV3:
     """
-    Constructs a MobileNetV3 Large architecture from
-    `"Searching for MobileNetV3" <https://arxiv.org/abs/1905.02244>`_.
+    MobileNetV3 model from
```
Comment (datumbox) [inline torchvision/models/quantization/mobilenetv3.py:197]: Why did you make this change? Clarifying that this is the variant ("Large" in this case) is something we do in all similar docs of models.

File: torchvision/models/quantization/mobilenetv3.py [hunk 3]
```diff
@@ -194,18 +194,33 @@ def mobilenet_v3_large(
...
+    `Searching for MobileNetV3 <https://arxiv.org/abs/1905.02244>`_.
 
-    Note that quantize = True returns a quantized model with 8 bit
-    weights. Quantized models only support inference and run on CPUs.
-    GPU inference is not yet supported
+    .. note::
+        Note that ``quantize = True`` returns a quantized model with 8 bit
+        weights. Quantized models only support inference and run on CPUs.
+        GPU inference is not yet supported
```
Comment (NicolasHug) [inline torchvision/models/quantization/mobilenetv3.py:203]: Thanks for making this a `note`, I agree this looks better this way. Would you like to open an issue to address the rest of the models? Or perhaps you'd like to submit a PR directly?

File: torchvision/models/quantization/mobilenetv3.py (hunk 2 above)
Comment (NicolasHug) [inline torchvision/models/quantization/mobilenetv3.py:197]:
```suggestion
    MobileNetV3 (Large) model from
```

**Review by NicolasHug — APPROVED** [review]
Looks great @abhi-glitchhg , thank you!

I adressed @datumbox 's comment above directly, so I'll merge now. <o/

File: torchvision/models/quantization/mobilenetv3.py (hunk 3 above)
Comment (abhi-glitchhg) [reply torchvision/models/quantization/mobilenetv3.py:203]: On my way to submit PR :)

**Pull request merged**